histomicstk/cli/BackgroundIntensity/BackgroundIntensity.xml

```diff
@@ -60,12 +60,13 @@
       <description>Use instead of sample_fraction to specify roughly how many pixels to sample.  The fewer tiles are excluded, the more accurate this will be.</description>
       <default>-1</default>
     </integer>
-    <double-vector>
-      <name>BackgroundIntensity</name>
-      <label>Background Intensity</label>
+    <file fileExtensions=".anot" reference="sample_slide_path">
```

Comment:
The reference has to match the name of the input file's parameter (`slide_path` in this case)